bot/src/main/java/com/almightyalpaca/discord/jdabutler/commands/commands/NotifyCommand.java

```diff
@@ -52,6 +53,24 @@ public void dispatch(final User sender, final TextChannel channel, final Message
             }
             return;
         }
+        else if(content.equalsIgnoreCase("list"))
+        {
+            EmbedBuilder embed = new EmbedBuilder();
+            embed.setDescription("You can subscribe to one of the following items. To unsubscribe simply type the command again.");
+            VersionCheckerRegistry.getVersionedItems().stream()
+                .filter(item -> item.getAnnouncementRole() != null)
+                .map(item ->
+                    new MessageEmbed.Field(
+                        item.getName().toUpperCase(),
+                        item.getDescription() +
+                            "\nCommand: `!notify " + item.getName().toLowerCase() + "`",
+                        false
+                    )
+                ).forEach(embed::addField);
+
+            channel.sendMessage(embed.build()).queue();
```

Comment:
can you use the respond() method in order to link the message for deletion?

Reply:
Should it really be deleted if it provides a list of information that you are actually supposed to read?

Reply:
it only gets deleted if the user deletes the command itself and i think thats fair

Reply:
```java
origMsg.getChannel().sendMessage(newMessageContent).queue(responseMsg ->
{
    responseMsg.delete().queueAfter(10, TimeUnit.SECONDS);
    origMsg.delete().queueAfter(10, TimeUnit.SECONDS);
});
```

This looks like it always deletes after 10 seconds

Reply:
Do you mean `reply(event, embed)`?

Reply:
yea, reply i guess. should only add to a LongLongMap